crates/sui-bridge/src/eth_syncer.rs

```diff
@@ -119,7 +119,7 @@ where
         contract_address: EthAddress,
         mut start_block: u64,
         mut last_finalized_block_receiver: watch::Receiver<u64>,
-        events_sender: mysten_metrics::metered_channel::Sender<(EthAddress, Vec<EthLog>)>,
+        events_sender: mysten_metrics::metered_channel::Sender<(EthAddress, u64, Vec<EthLog>)>,
```

Comment:
Maybe block height used for start_block and end_block deserve a new type, instead of just using u64.

Reply:
good point. I will add a TODO and convert all of them